src/test/java/de/hpi/datastreams/consumer/MyTest.java

```diff
@@ -1,37 +1,23 @@
 package de.hpi.datastreams.consumer;
 
-import de.hpi.datastreams.Main;
-import de.hpi.datastreams.messages.KeyRange;
-import de.hpi.datastreams.messages.Message;
-import de.hpi.datastreams.serialization.JSONSerde;
-import org.apache.kafka.clients.consumer.ConsumerRecord;
-import org.apache.kafka.clients.producer.ProducerRecord;
 import org.apache.kafka.common.serialization.StringDeserializer;
-import org.apache.kafka.common.serialization.StringSerializer;
 import org.apache.kafka.streams.StreamsBuilder;
 import org.apache.kafka.streams.TopologyTestDriver;
-import org.apache.kafka.streams.test.ConsumerRecordFactory;
-import org.apache.kafka.streams.test.OutputVerifier;
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-
-import static org.junit.Assert.assertNull;
-
 public class MyTest {
```

Comment:
If we want to stick with tests in the future, we might want to fix them. Otherwise, we can remove them.

Reply:
We should probably add some tests in the future.
But there is not that much to test right know (just dummy data/ messages).